base_layer/core/src/proof_of_work/monero_rx.rs

```diff
@@ -23,13 +23,16 @@
 use crate::{blocks::BlockHeader, proof_of_work::Difficulty};
 use bigint::uint::U256;
 use derive_error::Error;
+#[cfg(feature = "monero_merge_mining")]
```

Comment:
You can save some effort and just have this entire file behind the flag in `mod.rs`

Reply:
Will do